src/core/sys/posix/sys/ioctl.d

```diff
@@ -339,6 +333,11 @@ else version (OSX)
         ushort ts_yyy;
     }
 
+    enum uint TIOCGSIZE = TIOCGWINSZ;
+    enum uint TIOCSSIZE = TIOCSWINSZ;
+
+    import core.sys.posix.sys.filio; // File related ioctls
```

Comment:
Yes, that is what I intended -- that anyone wanting say the 'struct winsize' to do an ioctl would include both ttycom.d and ioctl.d mimicing what they would do in C on OSX (and think all the BSDs).